### DISCUSSION TITLE
am  i stupid ?

### DISCUSSION THREAD

**Discussion by 5RoD** [stefvanschie/IF#1846] [General]
![image](https://github.com/user-attachments/assets/ddcb493e-41b4-4e5a-ae33-f5fe8f059c4a)

how do i use this FRAMEWORK BRO

**Comment by stefvanschie** [comment]
Your code is directly inside the class and not within a method, constructor, or some other kind of code block. Statements like `gui.addItem(guiItem)` are not allowed there, only field declarations (plus initialisers). I'd recommend moving your code inside a method. Then you can call that method whenever you need to create a new gui.
